### PR TITLE
Update web3js.md

### PR DESCRIPTION
**Pull request by Vikash-8090-Yadav**
Update Read and Write function a/c to the smart contract written in the docs.

## Pre-flight checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://doc.confluxnetwork.org/docs/general/CONTRIBUTING#create-a-pull-request).
- [x] A relating issue is created: # (<- fill the issue number here)
- [x] **This is NOT a PR to submit translation**

### CODE REVIEW

File: docs/espace/tutorials/sdks/web3js.md [hunk 1]
```diff
@@ -1,4 +1,3 @@
----
```
Comment (0xn1c0) [inline docs/espace/tutorials/sdks/web3js.md:1]: This line shouldn't be removed. It makes the metadata get displayed on the document.
image

The other updates are correct.

**Review by 0xn1c0 — CHANGES_REQUESTED** [review]
Requested changes, first line was removed making the document display the metadata and should be fixed. 
`---`

**Review by 0xn1c0 — DISMISSED** [review]

File: docs/espace/tutorials/sdks/web3js.md (hunk 1 above)
Comment (Vikash-8090-Yadav) [reply docs/espace/tutorials/sdks/web3js.md:1]: Oh am really sorry. Correcting Now!!

**Review by Vikash-8090-Yadav — COMMENTED** [review]
@0xn1c0  Done. Can u please check once.

**Pull request merged**